cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilderTest.java

```diff
@@ -1426,6 +1426,101 @@ public void testWithExternalListenersIngress()  {
                 "listener.name.external-9094.ssl.keystore.type=PKCS12"));
     }
 
+    @ParallelTest
+    public void testWithExternalListenersClusterIPWithTLS()  {
```

Comment:
The rest of the code mostly uses strict camel case ⇒ `testWithExternalListenersClusterIpWithTls`

Reply:
cause of my golang developper practice and golint rules. terms like Id, Ip, Http are always used as all capital letter form (ID, IP, HTTP) in a composite function/type/var name
I can change it if it does not respect the project rules.